sis_import_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh --
+
+## find the next Jenkins build number
+## in a file named nextBuildNumber two parent directory up
+parentdir="$(dirname "$(pwd)")"
```

Comment:
Jenkins provides this in an environment variable: BUILD_NUMBER

The build script should work on a desktop too.  You won't have the BUILD_NUMBER variable then but you can default it to something like "local_build". The syntax is VARIABLE=${1:-DEFAULTVALUE}